### ISSUE TITLE
Use the node name as default for the wallet name

### ISSUE THREAD

**Issue by bota87** [LN-Zap/zap-android#256]
What about to use the node name as default for the wallet name instead of the host name?
I think this make a lot of sense for who have an onion address in the host name.

**Comment by michaelWuensch** [comment]
Hi @bota87,
thanks for your suggestion.
I have thought about this as well. The problem is the node name is not part of the lnd connect string and therefore not known in that phase.
The connection has to be successful before we can query the node name.
Now if we add a new wallet for example and there is some connection problem, it would have no name at all.

**Comment by bota87** [comment]
Hi @michaelWuensch,
I was sure there is a technical explanation behind this choice.
Could you think of using the url as a temporary name until the first connection could be a solution?
However this request is not that important so feel free to just close it
